test/unit/app/browser/api/ledgerTest.js

```diff
@@ -421,8 +424,14 @@ describe('ledger api unit tests', function () {
   })
 
   describe('transitionWalletToBat', function () {
+    let fakeClock
+
+    before(function () {
+      fakeClock = sinon.useFakeTimers()
```

Comment:
This is needed because code paths hitting setTimeout() will cause unit tests to crash (seemingly randomly) and in some cases hang.  When the fake timer is destroyed, all methods registered with setTimeout are cleaned up.